ui/app/reports/controllers/reportsController.js

```diff
@@ -50,6 +51,15 @@ angular.module('bahmni.reports')
             });
         };
 
+        var isPreviousMonth = function (date) {
+            return (new Date(date).getMonth() === new Date().getMonth() - 1 && new Date(date).getFullYear() === new Date().getFullYear())
```

Comment:
Can you add unit tests please that showcase the scenarios mentioned in JIRA ticket? Otherwise its hard to know if those use-cases will work, since we can't test those via the browser.